email_alert_notifications/jenkins.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+export FILE_TO_UPLOAD=rename_email_files_with_request_id.py.zip
+/usr/bin/zip -FS $FILE_TO_UPLOAD rename_email_files_with_request_id.py
```

Comment:
It might be simpler to delete the zip file and create a new one. `-FS` has some interesting other features.